js/index.js

```diff
@@ -40,3 +40,75 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+const codeImg = document.getElementById("cta-img");
+codeImg.setAttribute('src', siteContent["cta"]["img-src"])
+
+const midImg = document.getElementById("middle-img");
+midImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])
+
+document.querySelector('a:nth-of-type(1)').textContent = siteContent["nav"]["nav-item-1"]
```

Comment:
A good thing to remember is when you see code that is very similar and we are just basically copying and pasting.
This is is just copypasta, were copying code over and over again. It works, but it is not very scalable. This is what they might consider as a `code smell`. When things are easy to spot and can lead to deeper issues now or later on.